src/cloudWatchLogs/timeFilterSubmenu.ts

```diff
@@ -107,7 +132,7 @@ export class TimeFilterSubmenu extends Prompter<TimeFilterResponse> {
 
                         return { start: startTime.valueOf(), end: endTime.valueOf() }
                     } else {
-                        return resp
+                        return undefined
```

Comment:
This scenario triggers if the user cancels/exits. So we do not want to return anything, even the previous value since it triggers a reload.